### PR TITLE
Add External sources popover instead of the Meta search form

### LINKED ISSUE #1197: Rename "Meta Search" to "External Sources"
## Problem

We currently use two names "Meta Search" and "Additional Sources" to refer to the same feature: letting users search a list of external search engines that also contain CC-licensed content. I think "Additional Sources" is much more sensible for non-technical users. Additionally, using the word "Meta" in the context of our name "Openverse", as it relates to the "Metaverse" is also quite unfortunate.

## Description

Let's remove all references to 'meta search' and replace them with 'additional sources', including internal naming of code, methods, string keys, and so on.

We need to make sure a redirect is in place to preserve url so that `https://wordpress.org/openverse/meta-search/` redirects to `https://wordpress.org/openverse/additional-sources/`.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1197 by @panchovm

## Description

This PR implements the 2 changes from the linked issue:
- Moves the external sources buttons to a popup
- Renames the section of the `ContentSwitcher` from _Additional Sources_ to _Coming Soon_

## Testing Instructions

Run the app and check the _meta search_ section in the bottom of an image or an audio search page. It should have a popup button with links to external sources of openly-licensed media.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VButton.vue
Comment (obulat) [inline src/components/VButton.vue:5]: Making the button `inline-flex` makes both `button` and `a` elements behave similarly. Without it, by default, an `<a>` button takes up all of the parent's width.

File: tsconfig.json
Comment (obulat) [inline tsconfig.json:64]: Cannot add `VMetaSearch` to `tsconfig` anymore because it depends on Popover and Reakit.

**Comment by krysal** [comment]
Was #1205 too far behind to recover some of the changes from there?

**Comment by obulat** [comment]
> Was #1205 too far behind to recover some of the changes from there?

It was renaming the 'meta-search' term inside the code, and this PR should probably use some of the changes from there. Here, most of the changes are related to moving the Additional/External sources buttons to a popup.

**Comment by krysal** [comment]
If it is too cumbersome now, we can leave it as it is, but it would have been good to resolve that PR (update it or close it) before opening another related one.

Tagging @panchovm as he probably wants to check this!

**Comment by panchovm** [comment]
Thanks for tagging me @krysal 

I noticed two things:
## Popover padding
The popover has a `12px` padding instead of `8px` as [the mockup](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1393%3A20471). Below a screenshot of what I see.

CleanShot 2022-08-15 at 14 58 37@2x

## Popover description padding
The title and description area, both above the links list, have a `12px` for left and right padding instead of `16px`.

CleanShot 2022-08-15 at 14 59 51@2x

Related questions
On the other hand, I have two doubts.
* I do not see the "coming soon" section in the content switcher popover. Do we need to wait for making the 3D model support visible?
* This PR shows a change in the links section. Are we tackling the text changes in a different PR? I am referring to `Not finding what you're looking for? Try additional image sources` and `Click on a source below to directly search other collections of CC-licensed images` texts.

Despite the above, the popover looks great ✨

**Comment by obulat** [comment]
I've tried to update the paddings, @panchovm. Does it look good?
Screen Shot 2022-08-19 at 12 24 51 PM

I will also edit the text after #1647 is merged.

> I do not see the "coming soon" section in the content switcher popover. Do we need to wait to make the 3D model support visible?

This section is currently under a flag that is turned off in production.

**Review by panchovm — COMMENTED** [review]
It looks awesome ✨ 

I plan to share soon the idea we have discussed in parallel for the content switcher popover. Basically, removing the style of the "coming soon" area and making items more simple. Just mentioning it as something attached to the metasearch improvements (#1197)

Although I wonder if in this PR we should tackle the term replacement, changing from "additional source" to "external source" as proposed in this [idea's core](https://github.com/WordPress/openverse-frontend/issues/1197#issuecomment-1195901505).

Should we address that change here?

File: src/locales/en.json
Comment (zackkrida) [inline src/locales/en.json:513]:
```suggestion
    "caption": "{openverse} can not guarantee the accuracy of license information. Always verify that the work is actually under a CC license.",
```

"Openverse has no control" sounds aggressive to me. We can soften this language a little.

**Review by zackkrida — APPROVED** [review]
This looks fantastic, I made one suggestion concerning the language but LGTM once that is resolved.

**Comment by zackkrida** [comment]
@obulat looks like some new test failures here.

**Review by panchovm — APPROVED** [review]
It looks great ✨ 

I forgot that issue #734 was created to update this section. It is currently blocked as it was waiting for agreement on the "external source" term. Now that we made a decision and described the changes in [this comment](https://github.com/WordPress/openverse-frontend/issues/1197#issuecomment-1195901505), we can close it.

On the other hand, the metasearch design changes from a dropdown menu to a modal on the smallest breakpoint. Nonetheless, approving this PR but mentioning the modal feature as pending to tackle it in the future.

Great work @obulat

**Pull request merged**
